source/guide_flatboard.rst

```diff
@@ -0,0 +1,190 @@
+.. highlight:: console
+
+.. author:: Christian Kantelberg <uberlab@mailbox.org>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/flatboard.png
+      :align: center
+
+#########
+Flatboard
+#########
+
+Flatboard_ is a simple, lightweight, modern and fast flat-file forum, using Json and Markdown or BBcode.
```

Comment:
The control to select markdown instead of the default BBcode is actually disabled. Do you know why? :sweat_smile:   
![image](https://user-images.githubusercontent.com/2158203/53734991-df556980-3e85-11e9-807f-64e39b1516be.png)

Reply:
For the moment Markdown is not available, but should be fixed with one of the next updates.
<img width="892" alt="bildschirmfoto 2019-03-04 um 15 27 19" src="https://user-images.githubusercontent.com/8388140/53739465-192c6d00-3e92-11e9-81a8-ea41c321b62c.png">